workers/unity/Packages/com.improbable.gdk.tools/GdkToolsConfigurationEditorExtensions.cs

```diff
@@ -0,0 +1,154 @@
+﻿using System;
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+using Object = UnityEngine.Object;
+
+namespace Improbable.Gdk.Tools
+{
+    /// <summary>
+    ///     Defines a custom inspector window that allows you to configure the GDK Tools.
+    /// </summary>
+    [CustomEditor(typeof(ScriptableGdkToolsConfiguration))]
+    public class GdkToolsConfigurationInspector : Editor
+    {
+        internal const string SchemaStdLibDirLabel = "Schema Standard Library Directory";
+        internal const string CodegenOutputDirLabel = "Code Generator Output Directory";
+        internal const string SchemaSourceDirsLabel = "Schema Source Directories";
+
+        private const string CodeGeneratorLabel = "Code Generator Options";
+        private const string DownloadCoreSdkLabel = "Core SDK Options";
+
+        private const string AddSchemaDirButtonText = "Add Schema Source Directory";
+        private const string RemoveSchemaDirButtonText = "Remove";
+
+        private const string ResetConfigurationButtonText = "Reset GDK Tools Configuration to Default";
+
+        private ScriptableGdkToolsConfiguration toolsConfig;
+        private List<string> configErrors = new List<string>();
+
+        private readonly GUIStyle errorLayoutOption = new GUIStyle();
+
+        private void OnEnable()
+        {
+            if (toolsConfig != null)
+            {
+                return;
+            }
+
+            toolsConfig = ScriptableGdkToolsConfiguration.GetOrCreateInstance();
+
+            errorLayoutOption.normal.textColor = Color.red;
+
+            Undo.undoRedoPerformed += () => { configErrors = toolsConfig.Validate(); };
+        }
+
+        public override void OnInspectorGUI()
+        {
+            EditorGUI.BeginChangeCheck();
+
+            GUILayout.Label(DownloadCoreSdkLabel);
```

Comment:
You could clone the label skin and give it a right padding of `5` rather than having to space it explicitly.